spec/fixtures/container_registry/docker-credentials.json

```diff
@@ -0,0 +1 @@
+{"auths":{"registry.digitalocean.com":{"auth":"secret"}}}
```

Comment:
Could you add a newline add the end of this file to make it consistent with the others?